lib/BuildSystem/BuildSystemFrontend.cpp

```diff
@@ -232,12 +232,21 @@ class BuildSystemFrontendExecutionQueueDelegate
 };
 
 struct BuildSystemFrontendDelegateImpl {
+
+  /// The status of delegate.
+  enum class Status {
+    Uninitialized = 0,
```

Comment:
Don't think we need a state for that, we already fail and return errors in case of failed initialization.
https://github.com/apple/swift-llbuild/blob/master/lib/BuildSystem/BuildSystemFrontend.cpp#L546